code/modules/mob/living/simple_animal/abnormality/he/red_shoes.dm

```diff
@@ -0,0 +1,409 @@
+#define STATUS_EFFECT_REDPOSSESS /datum/status_effect/red_possess
+/mob/living/simple_animal/hostile/abnormality/red_shoes
+	name = "Red Shoes"
+	desc = "A pair of elegant red women's shoes. The design is antique, but there is no telling where and how they were made."
+	health = 800
+	maxHealth = 800
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "redshoes"
+	icon_living = "redshoes"
+	icon_dead = "redshoes_breach"//dels on death if it's possessing someone. Egg goes here
+	can_breach = FALSE
```

Comment:
This needs to be true if you intend for the pink midnight breach to work. Consider putting early returns under BreachEffect() so it only fires when you want.